deployment/templates/fluentd.yaml

```diff
@@ -84,6 +84,9 @@ objects:
             - name: dockerhostname
               mountPath: /etc/docker-hostname
               readOnly: true
+            - name: throttle-settings
```

Comment:
this is a host mount looking for a directory on the local host... i think it needs to be a secret? or does this DC declaration get overwritten by the secret volume later in deployment?